plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/conf/hydra/sweeper/ax.yaml

```diff
@@ -20,7 +20,7 @@ hydra:
       # Minimize objective:
       #   true: minimize
       #   false: maximize
-      minimize: false
+      minimize: true
```

Comment:
update the docs.

Reply:
https://github.com/facebookresearch/hydra/pull/434/commits/183d62ec8b20384c3bffb638e912fb4a0d421e66